app/ui/browser-modern/sagas/page-sagas.js

```diff
@@ -137,3 +141,8 @@ function* capturePage({ pageId, webview }) {
 
   yield put(ProfileEffects.addCapturedPage(pageId, readerResult));
 }
+
+function* getCertificateError({ pageId, url }) {
+  const { error, certificate } = yield Certificate.getCertificateError(url);
```

Comment:
This should be `yield call(Certificate.getCertificateError, url)`. I'm surprised this works as is.